pkg/messageformat/node.go

```diff
@@ -1,32 +1,61 @@
 package messageformat
 
+type AST []Node
```

Comment:
This does not enforce the type but it explains the intent. Ok for now.

Reply:
My bad. I thought it would